### ISSUE TITLE
line-shaped artifacts in reconstructed scenes

### ISSUE THREAD

**Issue by AnarchistKnight** [facebookresearch/projectaria_tools#156]
Good day.

I am trying to reconstruct and render scemes in AriaSynthesisDataset. The left image below is one that uses the complete sequence. Considering some sequences contain over 1500 frames, I sample every 3 frames to reduce time consumption. The right image below is the result. The line-shaped artifacts are more obvious than the one that builds from all frames in the sequence. I guess, building from more frames in some degree averages out this difference in brightness. I removed vignetting ([vignetting mask here](data/vignetting_mask/rgb_vignetting_half_reso_mask.png)) before reconstructing, subsequent frames still differ in brightness. Any suggestion to remove or reduce the artifacts?

![image](https://github.com/user-attachments/assets/5dc475ad-ae15-48db-a9e8-fcaaaeb064d4)
![image](https://github.com/user-attachments/assets/bf70ee45-08e8-433a-91ef-c123b500e8eb)

**Comment by suvampatra** [comment]
Please have a look at the discussion here  https://github.com/facebookresearch/projectaria_tools/issues/99

and this pull request which has a demo https://github.com/facebookresearch/projectaria_tools/pull/125

**Comment by AnarchistKnight** [comment]
> Please have a look at the discussion here #99
> 
> and this pull request which has a demo #125

Do you think I use the wrong vignetting mask?

**Comment by Rock-100** [comment]
@AnarchistKnight I think the masks here (https://github.com/facebookresearch/projectaria_tools/tree/main/data/vignetting_mask) are not for ASE dataset, as we can see their sizes are not the same with images in ASE. @suvampatra Could you please tell me how to find a proper mask?

**Comment by ChenYutongTHU** [comment]
> @AnarchistKnight I think the masks here (https://github.com/facebookresearch/projectaria_tools/tree/main/data/vignetting_mask) are not for ASE dataset, as we can see their sizes are not the same with images in ASE. @suvampatra Could you please tell me how to find a proper mask?


I agree with this. The image size in ASE is (704,704). However only vignette masks of (1408,1408) and (2816,2816) are provided.

Where can we find the vignette masks for ASE?

Thanks

**Comment by VladPyatov** [comment]
The release of the vignette mask for ASE dataset would be very helpful
